### PR TITLE
Refactor commandBye to commandQuit

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -210,29 +217,54 @@ public void displayOneProperty(Property property, int i) {
...
+    public void displayOnePropertyShort(Property property, int i) {
+        System.out.print(i + FULL_STOP);
+        System.out.println(SPACE + PROPERTY_ADDRESS_LABEL + property.getPropertyAddress());
+        System.out.println(SPACE + PROPERTY_UNIT_TYPE_LABEL + property.getUnitType());
+        System.out.println(SPACE + PROPERTY_RENTAL_LABEL + property.getRentingPrice());
+        System.out.println(SPACE + LINE_BREAK);
+    }
+
+    public void displayNoOfProperties(int i) {
+        if (i == 1) {
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:253]: Can consider replacing it with a boolean variable to make it more understandable for the reader.

File: src/main/java/seedu/duke/Ui.java [hunk 2]
```diff
@@ -210,29 +217,54 @@ public void displayOneProperty(Property property, int i) {
...
+            System.out.println("There is " + i + " property in the list");
+        } else {
+            System.out.println("There are " + i + " properties in this list");
+        }
+    }
+
+    public void displayNoOfClients(int i) {
+        if (i == 1) {
+            System.out.println("There is " + i + " client in the list");
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:262]: Can consider using the ui methods to show to user.

**Review by wilsonngja — APPROVED** [review]
Overall the code is good to merge. The minor discrepancies identified earlier has been corrected. Good job.

**Pull request merged**